file-upload/file-upload-scala-sbt/README.md

```diff
@@ -0,0 +1,66 @@
+# Lagom Recipe: How to upload a file
+
+It is not trivial how to handle file uploads in Lagom, Lagom is an abstraction over the transport and it doesn't include direct support for `mulipart-form-data`. 
+
+This recipe demonstrates how to fallback from [Lagom's `ServiceCall`](https://www.lagomframework.com/documentation/1.4.x/scala/ServiceImplementation.html#Implementing-services) implementation to it's building block: [Play Actions](https://www.playframework.com/documentation/2.6.x/ScalaActions#What-is-an-Action?). The final solution uses an [asynchronous Action](https://www.playframework.com/documentation/2.6.x/ScalaAsync#Make-controllers-asynchronous) so we can control what execution context is used to run the file handling logic.
+
+While this recipe focuses on solving the problem of handling file uploads you can learn from the solution presented here to fallback to Play Actions for other edge cases not supported by Lagom out of the box.
```

Comment:
I think we could shift the emphasis: it's not that Lagom doesn't support this out of the box: this _is_ how Lagom supports this. Anything we can do to reduce the perceived division between Lagom and Play will help people understand that the full capabilities of Play are available to any Lagom service.